app/Http/Controllers/SideMenu/SideMenuController.php

```diff
@@ -3,29 +3,21 @@
 namespace App\Http\Controllers\SideMenu;
 
 use App\Http\Controllers\Controller;
-use App\Settings\PathInfo;
 
-class SideMenuController extends Controller
-{
-    /*
-    |--------------------------------------------------------------------------
-    | SideMenuController
-    |--------------------------------------------------------------------------
-    |
-    | SideMenuコントローラ
-    |
-    */
+/**
+ * SideMenuController
+ * 
+ * ページを横にスワイプしたときに横から表示されるメニューを扱う
+ */
+class SideMenuController extends Controller {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition